Source/CronosPlayUnreal/Android/SecureStorage.java

```diff
@@ -0,0 +1,87 @@
+package com.cronos.play;
+
+import android.content.Context;
+import android.util.Log;
+import androidx.security.crypto.EncryptedFile;
+import androidx.security.crypto.MasterKey;
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.security.GeneralSecurityException;
+import java.io.ByteArrayOutputStream;
+import java.io.InputStream;
+import java.io.FileOutputStream;
+
+public class SecureStorage {
+
+    public static String readSecureStorage(Context context, String key) {
+        try {
+            MasterKey masterKey = new MasterKey.Builder(context)
+                    .setKeyScheme(MasterKey.KeyScheme.AES256_GCM)
+                    .build();
+
+            File file = new File(context.getFilesDir(), key);
+            EncryptedFile encryptedFile = new EncryptedFile.Builder(
+                    context,
+                    file,
+                    masterKey,
+                    EncryptedFile.FileEncryptionScheme.AES256_GCM_HKDF_4KB).build();
+
+            String result;
+            if (file.exists()) {
+                ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
+                InputStream inputStream = encryptedFile.openFileInput();
+                byte[] buffer = new byte[1024];
+                int len;
+                while ((len = inputStream.read(buffer)) != -1) {
+                    outputStream.write(buffer, 0, len);
```

Comment:
if larger than 1024 buffer, it'll throw an exception? for English mnemonics, I guess it's enough... but perhaps docs should mention the limitation in case developers tried to store some larger secret values?

Reply:
ok